src/Goerli-DssSpell.sol

```diff
@@ -46,19 +134,431 @@ contract DssSpellAction is DssAction {
+        // Increase WSTETH-B line from 200 million DAI to 500 million DAI
+        // Reduce WSTETH-B gap from 30 million DAI to 15 million DAI
+        DssExecLib.setIlkAutoLineParameters("WSTETH-B", 500 * MILLION, 15 * MILLION, 8 hours);
+        // Reduce ETH-B line from 500 million to 250 million DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("ETH-B", 250 * MILLION);
+        // Reduce WBTC-A line from 2 billion DAI to 500 million DAI
+        // Reduce WBTC-A gap from 80 million DAI to 20 million DAI
+        // Increase WBTC-A ttl from 6 hours to 24 hours
+        DssExecLib.setIlkAutoLineParameters("WBTC-A", 500 * MILLION, 20 * MILLION, 24 hours);
+        // Reduce WBTC-B line from 500 million DAI to 250 million DAI
+        // Reduce WBTC-B gap from 30 million DAI to 10 million DAI
+        // Increase WBTC-B ttl from 8 hours to 24 hours
+        DssExecLib.setIlkAutoLineParameters("WBTC-B", 250 * MILLION, 10 * MILLION, 24 hours);
+        // Reduce WBTC-C line from 1 billion DAI to 500 million DAI
+        // Reduce WBTC-C gap from 100 million DAI to 20 million DAI
+        // Increase WBTC-C ttl from 8 hours to 24 hours
+        DssExecLib.setIlkAutoLineParameters("WBTC-C", 500 * MILLION, 20 * MILLION, 24 hours);
+        // Reduce MANA-A line from 1 million DAI to 0 DAI
+        bytes32 _ilk = "MANA-A";
+        DssExecLib.removeIlkFromAutoLine(_ilk);
+        (,,, uint256 _line,) = VatLike(VAT).ilks(_ilk);
+        DssExecLib.setValue(VAT, _ilk, "line", 0);
+        DssExecLib.setValue(VAT, "Line", _sub(VatLike(VAT).Line(), _line));
+        // Reduce GUNIV3DAIUSDC1-A line from 1 billion DAI to 100 million DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("GUNIV3DAIUSDC1-A", 100 * MILLION);
+        // Reduce GUNIV3DAIUSDC2-A line from 1.25 billion DAI to 100 million DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("GUNIV3DAIUSDC2-A", 100 * MILLION);
+        // Reduce the UNIV2DAIUSDC-A line from 300 million DAI to 100 million DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("UNIV2DAIUSDC-A", 100 * MILLION);
+        // Reduce the PSM-USDP-A line from 500 million DAI to 450 million DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("PSM-PAX-A", 450 * MILLION);
+        // Reduce LINK-A gap from 7 million DAI to 2.5 million DAI
+        DssExecLib.setIlkAutoLineParameters("LINK-A", 5 * MILLION, 2_500_000, 8 hours);
+        // Reduce YFI-A gap from 7 million DAI to 1.5 million DAI
+        DssExecLib.setIlkAutoLineParameters("YFI-A", 3 * MILLION, 1_500_000, 8 hours);
+
+
+        // PSM tin increases
+        // Increase PSM-USDP-A tin from 0% to 0.1%
+        DssExecLib.setValue(MCD_PSM_PAX_A, "tin", 1 * WAD / 1000);
+        // Increase PSM-GUSD-A tin from 0% to 0.1%
+        DssExecLib.setValue(MCD_PSM_GUSD_A, "tin", 1 * WAD / 1000);
+
+        // PSM tout decrease
+        // Reduce PSM-GUSD-A tout from 0.2% to 0.1%
+        DssExecLib.setValue(MCD_PSM_GUSD_A, "tout", 1 * WAD / 1000);
```

Comment:
Couldn't fined where "tin" and "tout" changes are tested.
There is `checkPsmIlkIntegration` in the base tests, which you can use for that.

Reply:
Integration tests are added for this spell.